core/src/main/java/org/jdbi/v3/core/mapper/MapMappers.java

```diff
@@ -24,6 +24,7 @@
     private UnaryOperator<String> caseChange;
 
     public MapMappers() {
+        // TODO move to DefaultsPlugin because this is opinionated behavior
```

Comment:
I'm not sure what this comment means.  There has to be some default behavior without any plugins installed, and we're not likely to change it.  What would the default behavior be, if not the currently existing "lowercase"?

Reply:
The code-default ought to be NOP, no case-change. That stuff gets lowercased when put into a Map is a jdbi opinion. Sure surprised me when I used this feature for the first time, I was expecting the map to contain my columns with the name I had given them.